src/index.js

```diff
@@ -0,0 +1,75 @@
+import SauceTunnel from 'sauce-tunnel';
+import wd from 'wd';
+import promisify from 'es6-promisify';
+import { Promise } from 'es6-promise';
+

```

Comment:
the same as above about doubled empty lines. Add a line after `imports` block and `consts`